src/styles/utils/_extends.scss

```diff
@@ -1,4 +1,48 @@
-%h1 {
-  font-family: "Roboto", sans-serif;
-  font-weight: 400;
+%text-h2 {
+  font-size: 36px;
+  font-weight: 600;
+  line-height: 110%;
+  letter-spacing: 5%;
+  color: $secondary-color;
+  text-transform: uppercase;
+}
+
+%text-h4 {
```

Comment:
The font-family should be Sharp Sans No2 Medium or Avenir according to the task description.